### DISCUSSION TITLE
Typo in the first release?

### DISCUSSION THREAD

**Discussion by HarshPatel5940** [gofr-dev/gofr#184] [Q&A] [answered]
Congratulations for the first release with gofr! 

We might have done a typo in the release notes, where its states v1.0.0

![image](https://github.com/gofr-dev/gofr/assets/83941233/518d8c18-e845-4b88-aa96-07dee311ac74)

**Comment by HarshPatel5940** [comment]
I might have completely misunderstood why it's there, correct me if that's not an typo :D

**Reply by aryanmehrotra** [reply, marked as answer]
Thanks, I have fixed it.

**Comment by HarshPatel5940** [comment]
Great!
